extensions/notebook/src/jupyter/jupyterNotebookProvider.ts

```diff
@@ -55,28 +55,7 @@ export class JupyterNotebookProvider implements nb.NotebookProvider {
 	}
 
 	public get standardKernels(): nb.IStandardKernel[] {
-		return [
```

Comment:
This is not used any more. So removed it to avoid confusion.

Reply:
Excellent!